quetz/tests/api/test_api_keys.py

```diff
@@ -81,20 +81,8 @@ def test_list_keys_with_channel_roles(
     returned_keys = {key["description"]: key["roles"] for key in response.json()}
     assert len(returned_keys) == 2
 
-    assert returned_keys["user_is_user"] == [
-        {
-            "channel": private_channel.name,
-            "package": None,
-            "role": "maintainer",
-        }
-    ]
-    assert returned_keys["key"] == [
-        {
-            "channel": private_channel.name,
-            "package": None,
-            "role": "maintainer",
-        }
-    ]
+    assert returned_keys["user_is_user"] is None
+    assert returned_keys["key"] is None
 
 
 def test_list_keys_with_package_roles(
```

Comment:
this test is basically the same as the one before, also we don't need to add PackageMember/ChannelMember because we don't return the memberships with the api key any more